crates/router/src/services/authorization/roles.rs

```diff
@@ -78,7 +78,7 @@ impl RoleInfo {
             })
     }
 
-    pub async fn from_role_id_in_merchant_scope(
+    pub async fn from_role_id_in_lineage(
```

Comment:
Do we need this anymore? Should we just get the role by `role_id` and `org_id` while authorization.

Reply:
Yes we are using it for update and delete cases